src/replication/ng/flow/controller.go

```diff
@@ -55,7 +55,7 @@ type defaultController struct {
 	scheduler    scheduler.Scheduler
 }
 
-// Replicate according the to policy ID
+// Start a replication according to the policy
```

Comment:
// StartReplication starts a replication according to the policy

Reply:
It isn't a public function, so the comment should be ok